raiden/ui/cli.py

```diff
@@ -33,6 +33,7 @@
               'a keystore file exists in your local system.'),
         default=None,
         type=str,
+        envvar='ETH_ADDRESS',
```

Comment:
I think if we're already going for env var support it would be nicer to not special case this one parameter but rather use [click's `auto_envvar_prefix=`](http://click.pocoo.org/6/options/#values-from-environment-variables) support to automatically enable all parameters to be passed via env var.